modular_stonehedge/code/datums/traits/unspecial.dm

```diff
@@ -45,13 +46,13 @@
 	ADD_TRAIT(H, TRAIT_NOMOOD, QUIRK_TRAIT)
 
 /datum/quirk/value
-	name = "Appraiser"
+	name = "Skilled Appraiser"
 	desc = "I know how to estimate an item's value, more or less."
```

Comment:
it is not 'more or less' now, this could have been its own skill with more cost honestly.

Reply:
Honestly, being able to see how much mammon something is actually worth isn't worth extra cost.